crates/stackable-versioned-macros/src/attrs/item/mod.rs

```diff
@@ -240,13 +240,13 @@ impl CommonItemAttributes {
 
             // The convert_with argument only makes sense to use when the
             // type changed
-            if let Some(convert_func) = change.convert_with.as_ref() {
+            if let Some(upgrade_func) = change.upgrade_with.as_ref() {
```

Comment:
we need the same code block for downgrade pls

Reply:
Done in 4f3a8d8.